addon/services/cookies.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
-import getOwner from 'ember-getowner-polyfill';
 import _object from 'lodash/object';
 import _collection from 'lodash/collection';
 
-const { computed, computed: { reads }, isEmpty, typeOf, isNone, assert } = Ember;
+const {
+  computed,
+  computed: { reads },
+  inject: { service },
+  isEmpty,
+  typeOf,
+  isNone,
+  assert
+} = Ember;
 
 export default Ember.Service.extend({
-  _isFastBoot: reads('_fastBoot.isFastBoot'),
+  _isFastBoot: reads('fastboot.isFastBoot'),
 
-  _fastBoot: computed(function() {
-    let owner = getOwner(this);
-
-    return owner.lookup('service:fastboot');
-  }),
+  _fastBoot: service('fastboot'),
```

Comment:
This requires applications to have `ember-cli-fastboot` installed as well when using `ember-cookies` as this will fail when the `fastboot` service is not available - that's why I was using a regular `lookup` on the `owner` which will simply return `null` when the requested factory isn't available.